src/cow-react/pages/LimitOrders/index.tsx

```diff
@@ -1,25 +1,26 @@
 import * as styledEl from './styled'
 import {
   LimitOrdersWidget,
-  QuoteUpdater,
-  InfoPopup,
   MarketPriceUpdater,
+  InitialPriceUpdater,
   OrdersWidget,
   limitOrdersAtom,
 } from '@cow/modules/limitOrders'
 import { useAtomValue } from 'jotai/utils'
+import { ExecutionPriceUpdater } from '@cow/modules/limitOrders/updaters/ExecutionPriceUpdater'
```

Comment:
Shouldn't MarketPriceUpdater and InitialPriceUpdater be on `@cow/modules/limitOrders/updaters/ExecutionPriceUpdater` too?